src/App.js

```diff
@@ -3,9 +3,11 @@ import './App.css';
 import { RestaurantListItem } from './services/components/RestaurantListItem';
 import { fetchBusinesses } from './services/yelp';
 
-function App() {
+export default function App() {
   const [businesses, setBusinesses] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [zipCode, setZipCode] = useState([]);
+  const [search, setSearch] = useState([]);
```

Comment:
We would want the default state to be strings instead of arrays for zipCode and search.